### ISSUE TITLE
`ConVarExists` prints error when used on concommand without any feedback to lua

### ISSUE THREAD

**Issue by wrefgtzweve** [Facepunch/garrysmod-issues#6116]
Whenever `ConVarExists` is used on a console command it prints a error to console with no feedback to lua, code that likely causes it below. Ideally it errors or returns some other var. `ConcommandExists` could be nice too but that's not the issue here as it's just `ConVarExists` returning sorta "wrong" values.

Ideally it'd be nice to know if a cvar can be added, currently if `ConVarExists` returns false it cant always make a proper cvar resulting in issues down the line

![Image](https://github.com/user-attachments/assets/58d5dee2-c625-4da9-ae7e-1328704ac9b3)

Example code:
```lua
print( 1 )
concommand.Add( "example_cmd", function() end )
print( ConVarExists( "example_cmd" ) )
print( 2 )

-- CreateConVar( "example_cmd", 1 ) will error
```
Output:
```
1
Tried to look up command example_cmd as if it were a variable.
false
2
```

**Comment by robotboy655** [comment]
I don't think that this is wrong behavior. There is in fact no console variable with given name in your example. So the return value is correct.

`CreateConVar` will return `nil` (and produce a **non halting** error) when trying to make a convar with the name of a concommand. This is your feedback.
Unless you name your convars in a super generic pattern, this is really a non issue.

![Image](https://github.com/user-attachments/assets/d546f7d7-86e3-405b-a76c-8fd7798c63c1)
